spec/spec_helper.rb

```diff
@@ -93,6 +93,7 @@ def mock_login_gov(user, code = "ABC123") # rubocop:disable Metrics/AbcSize
   allow(login_gov).to receive(:exchange_token_from_auth_result).with(code).and_return(
     [{ email: user.email, sub: user.token }]
   )
+  allow(login_gov).to receive(:authorization_url).with(ial2: true)
```

Comment:
do we always want to mock this as true? seems like most users, specifically .gov and .mil users, will not be receiving `ial2=true` response from login.gov. is that correct?
i'm thinking maybe this should be a parameter of the `mock_login_gov` function, or added as a different mock function e.g. `mock_login_gov_ial2`. let me know if you disagree?

Reply:
Some of my specs were failing if I removed this line. Like that authorization_url method wasn't expecting to be able to receive ial2 as a param. I wasn't sure if this actually affected what was returned in the specs.